core/src/components/picker-column/test/standalone/index.html

```diff
@@ -31,7 +34,7 @@
         ['Tomato', 'Avocado', 'Onion', 'Potato', 'Artichoke'],
       ];
 
-      async function openPicker(numColumns = 1, numOptions = 5, columnOptions = defaultColumnOptions) {
+      async function openPicker(numColumns = 1, numOptions = 5, columnOptions = defaultColumnOptions, dynamic = false) {
```

Comment:
It might be tough to find this new test in the future since `standalone` is just for checking that basic functionality works without `ion-app`. Could we either move this to picker's `basic` test or create a new one entirely, and then add an E2E test to verify the behavior?

Reply:
+1 to this. My preference would be to create a separate E2E test so we can avoid screenshot diffs.